src/instructlab/configuration.py

```diff
@@ -109,16 +193,18 @@ class _chat(BaseModel):
     model_config = ConfigDict(extra="ignore")
 
     # required fields
-    model: StrictStr
+    model: str
 
     # additional fields with defaults
     vi_mode: bool = False
     visible_overflow: bool = True
     context: str = "default"
-    session: Optional[str] = None
-    logs_dir: str = "data/chatlogs"
+    session: typing.Optional[str] = None
+    logs_dir: str = Field(
+        default_factory=lambda: DEFAULTS.CHATLOGS
```

Comment:
this applies to other default dir names in config like MODELS: I'd recommend renaming to MODELS_DIR or something like that because DEFAULTS.CHATLOGS doesn't immediately make me think that the variable points to the chatlogs dir.

Reply:
non-blocking btw

Reply:
That makes sense, I think we can make that change.

Reply:
Resolved in commit https://github.com/instructlab/instructlab/pull/1471/commits/c8d73df3a538b99e2acd6d362bbbccacd82737eb